app/app.js

```diff
@@ -428,7 +428,7 @@ angular.module('portainer', [
       }
     })
     .state('service', {
-      url: '^/service/:id/',
+      url: '^/service/:id/:page_anchor',
```

Comment:
@eliat123 yes I agree but you do not need to add this as a state parameter. You can set the hash in the URL with `$location.hash(<HASH>)` and then call `$anchorScroll()` to automatically scroll to that hash.

See my comment below for a code example.